packages/core/src/components/popover/popover.tsx

```diff
@@ -422,6 +422,9 @@ export class Popover extends AbstractComponent<IPopoverProps, IPopoverState> {
     }
 
     private componentDOMChange() {
+        if (!this.targetElement) {
```

Comment:
We prefer `this.targetElement == null`, but we can merge and fix after the fact.